packages/api-plugin-fulfillment-method-shipping-flat-rate/src/getFulfillmentMethodsWithQuotesShippingFlatRate.js

```diff
@@ -0,0 +1,114 @@
+import Logger from "@reactioncommerce/logger";
+import isShippingRestricted from "./util/isShippingRestricted.js";
+import filterShippingMethods from "./util/filterShippingMethods.js";
+import { logCtx } from "./index.js";
+
+const packageName = "fulfillment-method-shipping-flat-rate";
+const fulfillmentTypeName = "shipping";
+const fulfillmentMethodName = "flatRate";
+
+/**
+ * @summary Returns a list of fulfillment method quotes based on the items in a fulfillment group.
+ * @param {Object} context - Context
+ * @param {Object} commonOrder - details about the purchase a user wants to make.
+ * @param {Array} [previousQueryResults] - an array of shipping rates and
+ * info about failed calls to the APIs of some shipping methods providers
+ * @returns {Array} - an array that contains two arrays: the first array will
+ * be an updated list of shipping rates, and the second will contain info for
+ * retrying this specific package if any errors occurred while retrieving the
+ * shipping rates.
+ * @private
+ */
+export default async function getFulfillmentMethodsWithQuotesShippingFlatRate(context, commonOrder, previousQueryResults = []) {
+  const { collections } = context;
```

Comment:
These two destructuring statements can be combined

Reply:
Fixed